config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  resources :wishlists
```

Comment:
In the controller you've provided only an action for `#new` and `#create`, so add only those route, not all at once.